LuckySpin/Services/SpinService.cs

```diff
@@ -19,7 +20,9 @@ public SpinService(ISpinRepository sr)
         public double CalculateAvgWins()
         {
             //TODO: Write logic to use the "real" spinRepository NOT the test data
-            return .1; 
+            double wins = spinRepository.GetSpins()
+           .Count(w => w.IsWinning == true);
+            return (wins + 1) / (spinRepository.GetCount() + 1);
```

Comment:
This assumption in line 25 is that the spin ALWAYS wins. While this is acceptable in our unit test code where we can handle each case (win or lose) with a different test, our service code has manage both.
Line 25 needs to find some way to check if the current spin is a winner.